feat: update docs to be more specific and reflect current features (#155)
open-sauced/docs: docs/community/highlights.md

  - highlights
---

> The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions, share the story, and inspire others to join you in your open source journey. For maintainers, it's a great way to showcase your project, the issues that need support, and attract new contributors.

Reviewer: [nitpick] I think some of the project links should be kept in this explanation. It could help people, especially contributors have some sort of idea on how they can start their journey. 
```suggestion
The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions, share the story, and inspire others to join you in your open source journey. For maintainers, it's a great way to showcase your project, the issues that need support, and attract new contributors.
```

Author: @CBID2 I didn't keep them because they don't reflect the implementation of the highlight.

If we want to include examples, it should be the PR, Issue, or Dev post.

> There are currently three types of highlights you can add to your profile: a Dev.to blog post, a pull request, or an issue.

Reviewer: [Proofreading] I think formatting them as a numbered list makes it easier for users and contributors to read. 
```suggestion
There are currently three types of highlights you can add to your profile: 1. a Dev.to blog post
2. a Pull request
3. a issue
```

![Example highlight](../../static/img/highlight.svg)

> For more examples of highlights, got to the [OpenSauced Insights feed](https://insights.opensauced.pizza/feed).

Reviewer: ```suggestion
For more examples of highlights, go to the [OpenSauced Insights feed](https://insights.opensauced.pizza/feed).
```
